src/features/game/lib/processEvent.ts

```diff
@@ -28,19 +27,21 @@ export const maxItems: Inventory = {
   "Rich Chicken": new Decimal("5"),
   "Fat Chicken": new Decimal("5"),
 
-  "Sunflower Seed": new Decimal(420),
-  "Potato Seed": new Decimal(220),
-  "Pumpkin Seed": new Decimal(170),
-  "Carrot Seed": new Decimal(120),
-  "Cabbage Seed": new Decimal(110),
-  "Beetroot Seed": new Decimal(100),
-  "Cauliflower Seed": new Decimal(100),
-  "Parsnip Seed": new Decimal(80),
-  "Radish Seed": new Decimal(60),
-  "Wheat Seed": new Decimal(60),
-  "Apple Seed": new Decimal(50),
-  "Blueberry Seed": new Decimal(50),
-  "Orange Seed": new Decimal(50),
+  // Seed limits + buffer of 30
+  "Sunflower Seed": new Decimal(510),
+  "Potato Seed": new Decimal(270),
+  "Pumpkin Seed": new Decimal(210),
+  "Carrot Seed": new Decimal(150),
+  "Cabbage Seed": new Decimal(138),
+  "Beetroot Seed": new Decimal(126),
+  "Cauliflower Seed": new Decimal(126),
+  "Parsnip Seed": new Decimal(102),
+  "Radish Seed": new Decimal(78),
+  "Wheat Seed": new Decimal(78),
+  "Kale Seed": new Decimal(66),
+  "Apple Seed": new Decimal(42),
+  "Orange Seed": new Decimal(42),
+  "Blueberry Seed": new Decimal(42),
```

Comment:
These fruit seeds decreased from 50 to 42, was this intended?

Reply:
It was 42 because I was keeping it consistent (+30 of inventory seed limit).  It was probably not necessary so I just reverted it back to 50